tests/spec/core/data-cite-spec.js

```diff
@@ -324,4 +324,47 @@ describe("Core — data-cite attribute", () => {
     ).toBeTruthy();
     expect(dahut).toBe(null);
   });
+
+  it("Adds <cite> around <a> when frag and path are missing from <dfn data-cite='some-spec'>", async () => {
```

Comment:
Can we merge this and the test below into a single test? Test boilerplate is larger than test itself 😅

Reply:
@marcoscaceres @pradeepgangwar

Reply:
agree.

Reply:
Okay, So should I remove this? @sidvishnoi

Reply:
oh okay so i should merge it into below test.